extension_src/api/utils.js

```diff
@@ -0,0 +1,22 @@
+import BrowserInterface from 'browser-interface';
+
+// When testing the app with `npm run serve`,
+// we don't have access to the WebExtension apis,
+// so we'll use a mock
+const useMock = window.location.href.includes('//localhost:');
+const driver = new BrowserInterface(useMock);
+
+const Utils = {
+  /*
+   * Get current page information Promise
+   */
+  getCurrentPage() {
+    return driver.tabs
+      .query({ active: true, lastFocusedWindow: true })
```

Comment:
A voir: est-ce que la fonction `driver.tabs.getCurrent()` ne serait pas plus appropriée ici ? https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/getCurrent

Reply:
> If you call it from a background script or a popup, it will return undefined.